src/Screens/components/MentorList.tsx

```diff
@@ -25,6 +26,9 @@ export default ({ onPress, testID }: Props) => {
   const userId = useSelector(tokenState.getUserId);
   const selectedSkills = useSelector(mentorState.getSelectedSkills);
 
+  const { hideInactiveMentors } = useSelector(
```

Comment:
In here you could add the property skillFilter in the destructuring and remove the selectedSkills from line 27

Reply:
Nice suggest

Reply:
```javascript
const { hideInActiveMentors, skillFilter } = selector
```

Reply:
Resolved